kotlin/add-graphics-renderer/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,4 @@
+<resources>
```

Comment:
this file looks unused and can be deleted

Reply:
you are correct!